utils/build/docker/java/install_dd_trace_api.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Would it make sense  to merge this content inside `utils/build/docker/java/install_ddtrace.sh` ? the purpose of this script is to handle binaries and ddtrce installation, so IMO, this logic should be kept inside it no ?

Reply:
Yep, the only problem is that the script must be called before compiling the weblog (to be able to install the api in the local maven repository), but I'll change it.

Reply:
Changed! :smile: